### ISSUE TITLE
bug: Ark SA Playerlist is empty

### ISSUE THREAD

**Issue by jammsen** [gamedig/node-gamedig#413]
**Describe the bug**
The Playerlist object is empty while numplayers are showing the right amount of players connected to the server

**Steps To Reproduce**
Tell me in discord to go on my server and use "npx gamedig --type asa jammsen.de:7783" on it.

**Expected behavior**
A not empty Playerlist object

**Comment by jammsen** [comment]
Ive added now quite a lot of "Test-Servers" to my site - https://jammsen.de/ - as you can see with the connect button you can get the ip, i tried almost all of them with "npx gamedig --type asa IP:PORT" and none return no players.

![image](https://github.com/gamedig/node-gamedig/assets/2270806/897548e8-f975-4467-81c4-79e161f52b40)

**Comment by podrivo** [comment]
I was testing some servers from this list: https://www.battlemetrics.com/servers/arksa. And the query is returning players inside `state.raw.totalPlayers`, which is also inside `state.numplayers`. As stated in the BattleMetrics page, `ARK: Survival Ascended does not support player lists.`

However, I've found [this Wiki](https://ark.wiki.gg/wiki/Server_Browser#Player_Entry) stating that it's possible to get the players list, but also mentions that it's using Valve's Protocol and not EOS, like it is currently. So I'm not sure if it's still true, as the page was last edited on 12 July 2022. Feels like this is related to the previous version of ARK, ARK: Survival Evolved, released in 2015.

If this is the case, I believe there's nothing to action here.

—

Query example:

```js
node bin/gamedig --type asa --pretty 207.244.82.105:5370

{
  "name": "EliteArk.com [Crossplay] 2man 10x Island1",
  "map": "TheIsland_WP",
  "password": false,
  "raw": {
    "deployment": "ad9a8feffb3b4b2ca315546f038c3ae2",
    "id": "0278a41cee444abb9ecf55887704c32e",
    "bucket": "TestGameMode_C:<None>:TheIsland_WP",
    "settings": {
      "maxPublicPlayers": 100,
      "allowInvites": true,
      "shouldAdvertise": true,
      "allowReadById": true,
      "allowJoinViaPresence": true,
      "allowJoinInProgress": true,
      "allowConferenceRoom": false,
      "checkSanctions": false,
      "allowMigration": false,
      "rejoinAfterKick": "",
      "platforms": null
    },
    "totalPlayers": 13,
    "openPublicPlayers": 87,
    "publicPlayers": [],
    "started": false,
    "lastUpdated": null,
    "attributes": {
      "MINORBUILDID_s": "47",
      "MODID_l": 0,
      "CUSTOMSERVERNAME_s": "EliteArk.com [Crossplay] 2man 10x Island1",
      "ISPRIVATE_l": 0,
      "SERVERPASSWORD_b": false,
      "MATCHTIMEOUT_d": 120,
      "ENABLEDMODSFILEIDS_s": "4923582,5016257,5024910,4894074,5029324,4928343,5031680,4923283,4888869,5009372,5036008,5033219",
      "DAYTIME_s": "1",
      "SOTFMATCHSTARTED_b": false,
      "STEELSHIELDENABLED_l": 1,
      "SERVERUSESBATTLEYE_b": true,
      "EOSSERVERPING_l": 162,
      "ALLOWDOWNLOADCHARS_l": 0,
      "OFFICIALSERVER_s": "0",
      "GAMEMODE_s": "TestGameMode_C",
      "ADDRESS_s": "207.244.82.105",
      "SEARCHKEYWORDS_s": "Custom",
      "__EOS_BLISTENING_b": true,
      "ALLOWDOWNLOADITEMS_l": 0,
      "LEGACY_l": 0,
      "ADDRESSBOUND_s": "207.244.82.105:5370",
      "SESSIONISPVE_l": 0,
      "__EOS_BUSESPRESENCE_b": true,
      "ENABLEDMODS_s": "931250,928793,928501,928708,940022,937546,939055,939228,928988,936882,930442,961348",
      "SESSIONNAMEUPPER_s": "ELITEARK.COM [CROSSPLAY] 2MAN 10X ISLAND1 - (V33.47)",
      "SERVERPLATFORMTYPE_s": "PC+PS5+XSX",
      "MAPNAME_s": "TheIsland_WP",
      "BUILDID_s": "33",
      "SESSIONNAME_s": "EliteArk.com [Crossplay] 2man 10x Island1 - (v33.47)"
    },
    "owner": "Client_xyza7891muomRmynIIHaJB9COBKkwj6n",
    "ownerPlatformId": null
  },
  "maxplayers": 100,
  "numplayers": 13,
  "players": [],
  "bots": [],
  "queryPort": 5370,
  "connect": "207.244.82.105:5370",
  "ping": 0
}
```

**Comment by jammsen** [comment]
I think i remember @CosminPerRam telling me in Discord that ASA uses EOS and that EOS (might) generally do not support giving playerlists out. I also never once successfully added an ASA server to the Steam-Server-Viewer also ASA isnt listed there as a game. Pretty sure that Valve thing was an ASE thing yeah @podrivo .

**Comment by podrivo** [comment]
Thank for the reply! Well, I guess this is safe to close then.

**Comment by jammsen** [comment]
If @CosminPerRam wants to close this, im fine with that.

**Comment by jammsen** [comment]
Sure, sounds nice, from my side this is already considered "done" 👍

**Comment by podrivo** [comment]
I'll make sure to add a note about this! (:

**Comment by CosminPerRam** [comment]
Closed by #489.
